images/releng/k8s-ci-builder/variants.yaml

```diff
@@ -5,11 +5,11 @@ variants:
     OS_CODENAME: 'bullseye'
```

Comment:
I think we should also switch the default config, right?

Reply:
1.17.8 is using bullseye.
Should we use `buster` for 1.18.1?